.github/workflows/dist.yml

```diff
@@ -4,17 +4,25 @@ on:
   push:
     tags:
       - "v[0-9]+.[0-9]+.[0-9]+"
+  workflow_dispatch:
 
 jobs:
   build:
     runs-on: ubuntu-latest
+    container:
+      image: registry.fedoraproject.org/fedora:latest
+      env:
+        PATH: /root/.local/bin:/root/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin
+    defaults:
+      run:
+        shell: bash
     steps:
+      - run: dnf install -y golang git meson 'python3dist(pip)' 'pkgconfig(yggdrasil)' 'pkgconfig(dbus-1)' 'pkgconfig(systemd)' jq ansible-core
```

Comment:
two small nits here to reduce the install base (and make this slightly faster):
- pass `--setopt install_weak_deps=False` to dnf (`dnf --setopt install_weak_deps=False install ...`) to not install recommended packages
- switch `git` with `git-core` to avoid the extra perl bits